playbooks/openshift-logging/private/config.yml

```diff
@@ -36,7 +36,7 @@
         --config={{ openshift.common.config_base }}/master/admin.kubeconfig
         get nodes
         -l {{ openshift_logging_es_nodeselector | map_to_pairs }}
-        -o jsonpath={.items[*].metadata.name}
+        -o jsonpath={.items[*].status.addresses[?(@.type==\"{{ __logging_node_type | default('Hostname') }}\")].address}
```

Comment:
This is the same approach I tried which allows as a dev to use node type to be 'InternalIP'.  Just a suggestion here but wouldnt a name like `__node_address_var` be more applicable?.  Other then that lgtm